script.js

```diff
@@ -0,0 +1,141 @@
+
+function firstFunction() {
+    console.log('True')}
+function secondFunction() {
+    console.log('False')}
+
+function isBoolTruthy(boolValue, firstFunction, secondFunction) {
+    if (boolValue) {
+        return firstFunction();
+    }
+    return secondFunction();
+}
+
+isBoolTruthy(true, firstFunction, secondFunction);
+
+// 5.After a hard quarter in the office you decide to get some rest on a vacation.
+// So you will book a flight for you and your girlfriend and try to leave all the mess behind you.
+// You will need a rental car in order for you to get around in your vacation.
+// The manager of the car rental makes you some good offers.
+// Every day you rent the car costs $40. If you rent the car for 7 or more days, you get $50 off your total.
+// Alternatively, if you rent the car for 3 or more days, you get $20 off your total.
+// Write a code that gives out the total amount for different days(d).
+
+function getTheTotalAmountForCarRental(dayNumber) {
+    const costPerDay = 40;
+    const fiftyOff = 50;
+    const twentyOff = 20;
+    if (dayNumber >= 7) {
+        return costPerDay * dayNumber - fiftyOff;
+    }
+    if (dayNumber >= 3) {
+        return costPerDay * dayNumber - twentyOff;
+    }
+    return costPerDay * dayNumber;
+}
+
+console.log(getTheTotalAmountForCarRental(6));
+
+// 6. Americans are odd people: in their buildings, the first floor is actually the ground floor and there is no 13th floor (due to superstition).
+// Write a function that given a floor in the american system returns the floor in the european system.
+// With the 1st floor being replaced by the ground floor and the 13th floor being removed, the numbers move down to take their place. In case of above 13, they move down by two because there are two omitted numbers below them.
+// Basements (negatives) stay the same as the universal level.
+
+function getTheFloorInTheEuropeanSystem(americanFloorNumber) {
+    if (americanFloorNumber <= 0) {
+        return americanFloorNumber;
+    }
+    if (americanFloorNumber >= 1 && americanFloorNumber <= 12) {
+        return americanFloorNumber - 1;
+    }
+    if (americanFloorNumber >= 14) {
+        return americanFloorNumber - 2;
+    }
+}
+
+console.log(getTheFloorInTheEuropeanSystem(14));
+
+// 7. In this kata, your task is to implement an extended version of the famous rock-paper-scissors game.
+
+function extendedVersionOfRockPaperScissors(playerOne, playerTwo){
+    if(playerOne === playerTwo) {
+        return 'Draw!';
+    }
+    if(
+        (playerOne === 'rock' && (playerTwo === 'scissors' || playerTwo === 'lizard')) ||
+        (playerOne === 'scissors' && (playerTwo === 'paper' || playerTwo === 'lizard')) ||
+        (playerOne === 'lizard' && (playerTwo === 'paper' || playerTwo === 'spock')) ||
+        (playerOne === 'paper' && (playerTwo === 'spock' || playerTwo === 'rock')) ||
+        (playerOne === 'spock' && (playerTwo === 'rock' || playerTwo === 'scissors'))
```

Comment:
That's super difficult to read.
Put the `extendedVersionOfRockPaperScissors` function in a separate JavaScript file.

Think about how you can split this function into multiple separate functions.
```
function willRockWin(opponent) {
  return opponent === 'lizard' || opponent === 'scissors';
}
```